Detectors/TRD/workflow/src/TrackBasedCalibSpec.cxx

```diff
@@ -133,14 +154,16 @@ DataProcessorSpec getTRDTrackBasedCalibSpec(o2::dataformats::GlobalTrackID::mask
                                                               o2::base::GRPGeomRequest::Aligned, // geometry
                                                               inputs,
                                                               true);
+  outputs.emplace_back(o2::header::gDataOriginTRD, "GAINCALIBHISTS", 0, Lifetime::Timeframe);
```

Comment:
The changes you do here to the DPL device should depend on whether the gain calibration was enabled or not. If not, then no additional inputs/outputs should be requested